trs/create_trs.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python3
+
+# This is a sample script to create or extend Riscure TRS files, using
+# the python library provided by Riscure.
+
+
+import random, os, trsfile
+from trsfile import trs_open, Trace, SampleCoding, TracePadding, Header
```

Comment:
could you add some info where to fetch that lib ?

Reply:
Hi, thanks for the response. I found it at github, but this package can be found in pip3, as well.

I have just created anew commit inserting this information in the head of the scripts which uses trsfile.
The github project is here:
https://github.com/Riscure/python-trsfile